.github/workflows/ci.yml

```
@@ -75,6 +75,10 @@ jobs:
         if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' && !inputs.skipTests }}
         run: make test-mk-v
 
+      - name: Run the tutorials
+        if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' && !inputs.skipTests }}
```

Could you add a specific input to the skip tutoriasl?
```suggestion
        if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' && !inputs.skipTutorials }}
```